🐞 Unable to `make build-egui` on macos

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

make fails on mac.

```
❯ make all
❯ make build-egui
docker build -t egui -f src\electionguard_gui\Dockerfile .
[+] Building 0.0s (1/2)                                                                                                                                  
 => [internal] load build definition from srcelectionguard_guiDockerfile                                                                            0.0s
 => => transferring dockerfile: 2B                                                                                                                  0.0s
failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount2318219980/srcelectionguard_guiDockerfile: no such file or directory
make: *** [build-egui] Error 1
```

### Expected Behavior

make succeeds on mac.

### Steps To Reproduce

...

### Environment

```markdown
- OS:
d
```


### Anything else?

d

--- a/Makefile
+++ b/Makefile
@@ -243,7 +243,7 @@ stop-db:
 	docker compose --env-file ./.env -f src/electionguard_db/docker-compose.db.yml down
 
 build-egui:
-	docker build -t egui -f src\electionguard_gui\Dockerfile .
+	docker build -t egui -f ./src/electionguard_gui/Dockerfile .
 
 start-egui: build-egui
 ifeq "${EG_DB_PASSWORD}" ""

